db/migrate/20180202191333_add_full_text_index_to_products.rb

```diff
@@ -0,0 +1,5 @@
+class AddFullTextIndexToProducts < ActiveRecord::Migration
+  def change
+    add_index :products, [:name, :description], type: :fulltext
```

Comment:
This index type is not valid for oracle. If we use the plain like search, does this make a difference on mysql?